Buddies/Activities/ActivityTableVC.swift

```diff
@@ -46,7 +46,7 @@ class ActivityTableVC: UITableViewController, FilterSearchBarDelegate {
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        fetchAndLoadActivities(for: lastSearchParam)
+        fetchAndLoadActivities(for: nil)
```

Comment:
@JakeThurman what's this do?

Reply:
Tells us not to use the last params we had but to fetch the current search params.

Reply:
ahh cool